.gitignore

```diff
@@ -0,0 +1,11 @@
+# OS generated files
+.DS_Store
+.DS_Store?
+._*
+.Spotlight-V100
+.Trashes
+Icon?
+ehthumbs.db
+Thumbs.db
+Desktop.ini
+$RECYCLE.BIN/
```

Comment:
Normally most people should have these in their global .gitignore, but it doesn't hurt to have them here as well.